pinot-common/pom.xml

```diff
@@ -340,12 +340,6 @@
         </plugins>
       </build>
     </profile>
-    <profile>
-      <id>presto-driver</id>
-      <properties>
-        <jdk.version>8</jdk.version>
```

Comment:
🎉

Reply:
The final goal is to remove the JDK8 build profile and move that to the presto dependency side :p